components/org.wso2.carbon.identity.oauth.dcr.endpoint/src/main/resources/api.identity.oauth.dcr.endpoint.yaml

```diff
@@ -27,13 +27,13 @@ paths:
   /register:
     post:
       description: |
-        This API is used to create an OAuth2 application.
+        This API is used to create an OAuth2 application. If your want to register your application using the implicit or authorization_code password grant types, you need to provide the redirect_uris data type in your API request.
+        If you are using the password grant type to sign your application, you don't need to use the redirect_uri data type. Check the example that is given below.
```

Comment:
any other grant type other than authorization_code and implicit (ie. Let's say password grant type)